CustomerJourney/package.json

```diff
@@ -26,7 +26,7 @@
     "lit-html": "1.2.1"
   },
   "peerDependencies": {
-    "@cjaas/common-components": "4.0.1",
+    "@cjaas/common-components": "4.1.1-dev1",
```

Comment:
just remember to update this to a real version, not a dev version once all is approved and merged.

Reply:
Sure.

Reply:
Sure